types/node/ts3.7/assert.d.ts

```diff
@@ -0,0 +1,57 @@
+declare module "assert" {
+    function internal(value: any, message?: string | Error): asserts value;
```

Comment:
Any reason this is called "internal" and not just `assert`? I wonder if TS auto-import can work better if it is named the way people usually import it...

Reply:
This also makes quick info for `assert` in `import assert = require("assert")` appear as `namespace internal (alias)`, which would be undesirable. I agree that this should be changed to `assert`.